security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -13,6 +13,24 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-59"]
+== OpenShift Compliance Operator 0.1.59
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.59:
+
+* link:https://access.redhat.com/errata/RHBA-2022:8538[RHBA-2022:8538 - OpenShift Compliance Operator bug fix update]
+
+[id="compliance-operator-0-1-59-bug-fixes"]
+=== Bug fixes
+
+* Previously, support for Payment Card Industry Data Security Standard (PCI DSS) and PCI DSS node did not show on `ppc64le`, which left profiles disabled in the architecture. Now, support for PCI DSS and PCI DSS node shows on `ppc64le` and enables all profiles in the architecture. (link:https://issues.redhat.com/browse/OCPBUGS-3252[*OCPBUGS-3252*])
+
+* Previously, after the recent upgrade to version 0.1.57 , the `rerunner` service account (SA) was no longer owned by the cluster service version (CSV), which caused the SA to be removed. Now, the CSV owns the `rerunner` SA in 0.1.57. (link:https://issues.redhat.com/browse/OCPBUGS-3452[*OCPBUGS-3452*])
+
+* Previously, after the recent upgrade to version 0.1.57, the metrics configuration default was 8080 and the `TargetDown` alarm was removed from the compliance operator. Now, the metrics configuration default is 8383 and the `TargetDown` remains in the compliance operator after 0.1.57. (link:https://issues.redhat.com/browse/OCPBUGS-3097[*OCPBUGS-3097*])
```

Comment:
Let's put backticks around the port numbers and also state that they are indeed port numbers for clarity:

was port `8080` 

... default is port `8383` 

... Compliance Operator after 0.1.57. (Always capitalize Compliance Operator, File Integrity Operator, etc)

Reply:
Updated to reflect suggested changes! :)